### PR TITLE
Code of conduct link fix

### PR DESCRIPTION
**Pull request by Hailfire805**
### Description

Just fixing a broken link within the readme.

Closes #

Code of conduct link doesn't give a 404

image
image

### CODE REVIEW

**Comment by Hailfire805** [comment]
I'd love to get feedback on if I did this right, It's my first pull request ever

File: README.md [hunk 1]
```diff
@@ -38,7 +38,7 @@ Refer to [CONTRIBUTING.md](./CONTRIBUTING.md).
 
 # Code of Conduct
 
-Before contributing to this repository, please read the [code of conduct](.master/CODE_OF_CONDUCT.md).
+Before contributing to this repository, please read the [code of conduct](CODE_OF_CONDUCT.md).
```
Comment (rizwanmustafa) [inline README.md:41]: Add `./` just to be safe.

```suggestion
Before contributing to this repository, please read the [code of conduct](./CODE_OF_CONDUCT.md).
```

**Review by rizwanmustafa — CHANGES_REQUESTED** [review]

**Comment by rizwanmustafa** [comment]
> Closes #

You don't need to post a screenshot of the code changes, everything else if fine.

**Review by rizwanmustafa — APPROVED** [review]

**Comment by Hailfire805** [comment]
> > Closes #
> 
> You don't need to post a screenshot of the code changes, everything else if fine.

Okay, should I still include them for things such as the 404 page?

**Comment by rizwanmustafa** [comment]
> Okay, should I still include them for things such as the 404 page? I also am not sure if there is anything else I need to do for this request, or do I just wait now.

The screenshot for 404 page is nice to have, it is visualization of the problem you solved. Now all you have to do is to wait for Miodec (Project Owner) or one of the collaborators to merge this PR (**P**ull **R**equest)

**Comment by rizwanmustafa** [comment]
Btw, thanks for solving this bug

**Review by Miodec — APPROVED** [review]
Thanks

**Pull request merged**